include/crm/common/alerts_internal.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2015 Andrew Beekhof <andrew@beekhof.net>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+#ifndef ALERT_INTERNAL_H
+#define ALERT_INTERNAL_H
+/* Default-Timeout to use before killing a notification script (in milliseconds) */
+#  define CRMD_NOTIFY_DEFAULT_TIMEOUT_MS (30000)
+
+/* Default-Format-String used to pass timestamps to the notification scripts */
+#  define CRMD_NOTIFY_DEFAULT_TSTAMP_FORMAT "%H:%M:%S.%06N"
+
+typedef struct {
+    char *name;
+    char *value;
+} envvar_t;
+
+typedef struct {
+    char *id;
+    char *path;
+    int timeout;
+    char *tstamp_format;
+    char *recipient;
+    GListPtr envvars;
+} notify_entry_t;
+
+enum notify_keys_e{
+    CRM_notify_recipient = 0,
+    CRM_notify_node,
+    CRM_notify_nodeid,
+    CRM_notify_rsc,
+    CRM_notify_task,
+    CRM_notify_interval,
+    CRM_notify_desc,
+    CRM_notify_status,
+    CRM_notify_target_rc,
+    CRM_notify_rc,
+    CRM_notify_kind,
+    CRM_notify_version,
+    CRM_notify_node_sequence,
+    CRM_notify_timestamp
+};
```

Comment:
Let's rename notify_keys_e to crm_alert_keys_e, and change the CRM_notify_ names to start with CRM_alert_.